inheritance/src/main/java/inheritance/McDonaldBallard.java

```diff
@@ -0,0 +1,10 @@
+package inheritance;
+
+public class McDonaldBallard extends McDonald {

```

Comment:
Again, another good choice to choose inheritance, because MacDonaldsBallard **_is a_** McDonalds